docs/quick-start.md

```diff
@@ -40,7 +40,7 @@ RDDs have _[actions](programming-guide.html#actions)_, which return values, and
 
 {% highlight scala %}
 scala> textFile.count() // Number of items in this RDD
-res0: Long = 126
+res0: Long = 99
```

Comment:
Oh I get it, it was already in the docs as 19 later. You're not saying you had a reproduction of a bug. OK yes of course it should be consistent at 19 (or whatever it is now).